src/document-click.js

```diff
@@ -35,7 +35,12 @@ import {urlReplacementsForDoc} from './url-replacements';
  * @param {!./service/ampdoc-impl.AmpDoc} ampdoc
  */
 export function installGlobalClickListenerForDoc(ampdoc) {
-  fromClassForDoc(ampdoc, 'clickhandler', ClickHandler);
+  registerServiceBuilderForDoc(
+      ampdoc,
+      'clickhandler',
+      ClickHandler,
+      /* opt_factory */ undefined,
+      /* opt_instantiate */ true);
```

Comment:
Nit: Note that this immediately instantiates the service in the method documentation.